### PR TITLE
docs: prepare stubs for documentation

### PR DESCRIPTION
**Pull request by perk-sumo**

### CODE REVIEW

**Comment by pmalek-sumo** [comment]
@perk-sumo Can we maybe add `docs/README.md` and link to it from the main `README.md`?

**Review by sumo-drosiek — APPROVED** [review]

File: docs/Configuration.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+Link to [Telegraf Receiver detailed configuration][telegrafreceiver_readme]
+
+### OTLP Receiver
+
+OTLP Receiver simple config
+
+Link to [OTLP Receiver detailed configuration][otlpreceiver_readme]
+
+## Processors
```
Comment (sumo-drosiek) [inline docs/Configuration.md:33]: What about processors :thinking:

**Review by astencel-sumo — APPROVED** [review]

File: docs/Configuration.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Exporters
+
+[filelogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.27.0/receiver/filelogreceiver
+[syslogreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.27.0/receiver/syslogreceiver
+[otlpreceiver_readme]: https://github.com/open-telemetry/opentelemetry-collector/tree/v0.27.0/receiver/otlpreceiver
```
Comment (kkujawa-sumo) [inline docs/Configuration.md:39]: Why not provide link to our fork https://github.com/SumoLogic/opentelemetry-collector?

Comment (sumo-drosiek) [reply docs/Configuration.md:39]: because we are using the contrib ones

Comment (perk-sumo) [reply docs/Configuration.md:39]: Yeah, I basically looked into the builder definition and used what is in there.

File: docs/Configuration.md (hunk 1 above)
Comment (perk-sumo) [reply docs/Configuration.md:33]: If we include any we should provide info how to configure them. It's a stub now.

**Review by pmalek-sumo — APPROVED** [review]

**Review by pmalek-sumo — APPROVED** [review]

**Pull request merged**